<?xml version="1.0" encoding="UTF-8"?>

<launch>
      <include file="$(find usb_cam)/launch/usb_cam-test.launch">
      </include>

      <include file="$(find robotic_arm_quark)/launch/load_param.launch">
      </include>

      <node name="controller" pkg="pid" type="controller" ns="CV_pidx" output="screen">
            <param name="node_name" value="sweep_pid" />
            <param name="Kp" value="5.0" />
            <param name="Ki" value="0.0" />
            <param name="Kd" value="0.1" />
            <param name="upper_limit" value="10" />
            <param name="lower_limit" value="-10" />
            <param name="windup_limit" value="10" />
            <param name="max_loop_frequency" value="100.0" />
            <param name="min_loop_frequency" value="100.0" />
      </node>

      <node name="controller" pkg="pid" type="controller" ns="CV_pidy" output="screen">
            <param name="node_name" value="sweep_pid" />
            <param name="Kp" value="5.0" />
            <param name="Ki" value="0.0" />
            <param name="Kd" value="0.1" />
            <param name="upper_limit" value="10" />
            <param name="lower_limit" value="-10" />
            <param name="windup_limit" value="10" />
            <param name="max_loop_frequency" value="100.0" />
            <param name="min_loop_frequency" value="100.0" />
      </node>

      <node name="CVclient" pkg="robotic_arm_quark" type="CVtest_client.py" output="screen">
      </node>

      <node name="CV" pkg="robotic_arm_quark" type="CVtest.py" output="screen">
      </node>

      <node name="CVserver" pkg="robotic_arm_quark" type="CVtest_server.py" output="screen">
      </node>

</launch>